src/ontodia/data/utils.ts

```diff
@@ -0,0 +1,29 @@
+import * as N3 from 'n3';
+import { RdfNode, Triple } from './sparql/sparqlModels';
+
+export function n3toRdfNode(entity: string): RdfNode {
+    if (N3.Util.isLiteral(entity)) {
+        return {type: 'literal', value: N3.Util.getLiteralValue(entity), 'xml:lang': ''};
```

Comment:
Use `N3.Util.getLiteralType()` and `N3.Util.getLiteralLanguage()` to extract both `datatype` and `xml:lang`.